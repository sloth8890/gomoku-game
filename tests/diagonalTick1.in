place A1
place A11
place B2
place M1
place C3
place S1
place D4
place R1
place E5
