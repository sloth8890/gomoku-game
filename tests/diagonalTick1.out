Black wins!
A1A11B2M1C3S1D4R1E5
Thank you for playing!